src/utils.js

```diff
@@ -208,3 +256,29 @@ export const findIndexByTimeStamp = (data, exactTimestamp) => (
       { value: currentValue, index: currentIndex } : { value, index }
   ), { value: 0, index: 0 }).index
 );
+
+export const calculateTimings = (pages) => (
+  pages.reduce(({ DOMContentLoaded, onLoad }, { pageTimings }) => ({
+    DOMContentLoaded: DOMContentLoaded + pageTimings.onContentLoad,
+    onLoad: onLoad + pageTimings.onLoad,
+  }), { DOMContentLoaded: 0, onLoad: 0 }));
+
+export const formatSize = (bytes) => {
+  if (bytes < 1024) {
+    return `${bytes}B`;
+  }
+  if (bytes < (1024 * 1024)) {
+    return `${Math.round(bytes / 1024)}KB`;
+  }
+  return `${Math.round((bytes / (1024 * 1024)))}MB`;
+};
+
+export const formatTime = (time) => {
+  if (time < 1000) {
+    return `${time}ms`;
```

Comment:
![Screen Shot 2020-01-07 at 2 31 12 PM](https://user-images.githubusercontent.com/39295560/71882559-b57e6a80-315a-11ea-9368-f282ff806e35.png)

can we use `Math.round` here?